### ISSUE TITLE
 Quantization Error

### ISSUE THREAD

**Issue by aspassiani** [felixriese/susi#39]
Hello, 

I think there might be an issue with the Quantization calculation in Susi. The error seems to be much large then when I attempt the calculation myself. I believe the line below is missing the axis being specified, (axis=1):

        quantization_errors = np.linalg.norm(
            np.subtract(weights_per_datapoint, X)
        ) 

Thanks,

**Comment by felixriese** [comment]
Hi @aspassiani, could you give me an example with small example data?

**Comment by aspassiani** [comment]
Hi @felixriese, not quite sure the best example to give, but if I were to set  weights_per_datapoint and X as random matrixes,

```
 weights_per_datapoint  =[[8,5,0,4,5,7,0,2,0,2]
                          [4,0,7,4,8,0,4,2,2,8]
                          [5,3,1,2,7,9,9,8,8,7]
                          [8,6,8,7,7,5,4,7,1,1]
                          [6,2,3,5,7,5,9,5,9,0]]

X = [[8,9,7,8,6,3,0,4,1,6]
    [3,2,1,2,5,7,0,0,3,3]
    [0,7,5,1,4,5,7,5,8,5]
    [9,1,1,7,5,9,8,9,3,3]
    [2,7,7,2,3,3,3,3,3,3]]
```

then,

    quantization_errors = np.linalg.norm(np.subtract(weights_per_datapoint, X)) 

would give me a single value of `25.729360660537214` making `return np.mean(quantization_errors)` unnecessary, whereas,

    quantization_errors = np.linalg.norm(np.subtract(weights_per_datapoint, X), axis = 1 ) 

would return a vector of values, `[10.9087,12.2066,10,11.0905,13.0767]` and a mean of `11.45650021348017`.

**Comment by felixriese** [comment]
@aspassiani Thanks for the data! Makes sense. I added a fix in the PR #40. Please have a look
